deploy-agent/deployd/common/caller.py

```diff
@@ -25,10 +26,15 @@ def __init__(self):
 
     @staticmethod
     def call_and_log(cmd, **kwargs):
+        output = ""
         try:
             process = subprocess.Popen(cmd, stdout=subprocess.PIPE,
                                        stderr=subprocess.PIPE, **kwargs)
-            output, error = process.communicate()
+            while process.poll() is None:
+                line = process.stdout.readline()
+                if line:
+                    output = output + "[%s]" % time.time() + line
```

Comment:
With this change, the time is not the actual time that the log was created but the current time. I think the right approach is to let the subprocess write timestamp to its stdout